plop_templates/test_template.txt

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import sinon from 'sinon';
+//import sinon from 'sinon';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
 
 import {{properCase name}} from '../index';
 
 describe('{{properCase name}}', () => {
-  let defaultProps;
+  // let defaultProps;
+  //
+  // beforeEach(() => {
+  //   defaultProps = {
+  //     // Add default props here
+  //   };
+  // });
```

Comment:
Left the comments on the template because they are important but want to pass linters on first pass.